packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -166,10 +166,32 @@ export default async function initSanity(
       selectedPlan = await getPlanFromCoupon(apiClient, intendedCoupon)
       print(`Coupon "${intendedCoupon}" validated!\n`)
     } catch (err) {
-      throw new Error(`Unable to validate coupon code "${intendedCoupon}":\n\n${err.message}`)
+      const useDefaultPlan = await prompt.single({
```

Comment:
Would it be a good idea to check whether the error is due to the coupon not existing, rather than some other failure (e.g. networking or backend)?